### PR TITLE
Add documentation for client policies

### PR DESCRIPTION
**Pull request by PatoBeltran**
Adding docs for client policies. This includes

- Changes to the config
- Errors and Warnings
- trusted-signers command

### CODE REVIEW

**Comment by karann-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/Errors-and-Warnings.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1156) | [Details](#user-content-docs%2Freference%2FErrors-and-Warnings.md)
[docs/reference/errors-and-warnings/NU3034.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3034.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3034?branch=pr-en-us-1156) | [Details](#user-content-docs%2Freference%2Ferrors-and-warnings%2FNU3034.md)
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3040.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3040.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3040?branch=pr-en-us-1156) |
[docs/reference/nuget-config-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1156) |
[docs/tools/cli-ref-trusted-signers.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/tools/cli-ref-trusted-signers.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-trusted-signers?branch=pr-en-us-1156) |

<a name="docs%2Freference%2FErrors-and-Warnings.md"></a>
### [docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/Errors-and-Warnings.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/reference/erorrs-and-warnings/NU3040.md).

<a name="docs%2Freference%2Ferrors-and-warnings%2FNU3034.md"></a>
### [docs/reference/errors-and-warnings/NU3034.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3034.md)
 - **Line 22**: **[Warning]** Illegal link: `&lt;a href=&quot;reference/nuget-config-file.md#trustedSigners-section&quot;&gt;trustedSigners&lt;/a&gt;` -- missing bookmark. The file reference/nuget-config-file.md doesn&#39;t contain a bookmark named trustedSigners-section.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811130041339106-1156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=G%2BjLIS7vmGDcGCch%2Bj5RA0PZIbgybLL3RIG4v3n0tWM%3D&st=2018-11-13T00%3A37%3A54Z&se=2018-12-14T00%3A42%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3034.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3034?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3040.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3040.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3040?branch=pr-en-us-1156) |
[docs/reference/nuget-config-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1156) |
[docs/tools/cli-ref-trusted-signers.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/tools/cli-ref-trusted-signers.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-trusted-signers?branch=pr-en-us-1156) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811131909596727-1156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xslXPn22BE%2BJ0C5yyHPr7m%2BcH13rb8yehPOUpyplA3U%3D&st=2018-11-13T19%3A06%3A13Z&se=2018-12-14T19%3A11%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3034.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3034?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3040.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3040.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3040?branch=pr-en-us-1156) |
[docs/reference/nuget-config-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1156) |
[docs/tools/cli-ref-trusted-signers.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/tools/cli-ref-trusted-signers.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-trusted-signers?branch=pr-en-us-1156) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811132304486233-1156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XO6QwNXoJgXiDI3RYBERIzVIi2OvPnSiPxtnP24nCyc%3D&st=2018-11-13T23%3A01%3A03Z&se=2018-12-14T23%3A06%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/nuget-config-file.md [hunk 1]
```diff
@@ -232,6 +235,35 @@ Identifies to the currently active source or indicates the aggregate of all sour
     <add key="All" value="(Aggregate source)" />
 </activePackageSource>
 ```
+## trustedSigners section
+
+Stores trusted signers used to allow package while installing or restoring. This list cannot be empty when the user sets `signatureValidationMode` to `require`.
```
Comment (rido-min) [inline docs/reference/nuget-config-file.md]: >Stores trusted signers used to allow package while installing or restoring

Does not sounds good to me, I'd rephrase to:

Stores the certificate fingerprint used to evaluate trust  when installing or restoring packages in `require` mode.

File: docs/reference/nuget-config-file.md [hunk 2]
```diff
@@ -308,5 +340,19 @@ Below is an example `nuget.config` file that illustrates a number of settings:
     <apikeys>
         <add key="https://MyRepo/ES/api/v2/package" value="encrypted_api_key" />
     </apikeys>
+
+    <!--
```
Comment (rido-min) [inline docs/reference/nuget-config-file.md]: Do we need XML Comments in the docs?

File: docs/tools/cli-ref-trusted-signers.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+## Usage
+
+```cli
+nuget trusted-signers <list|add|remove|sync> [options]
+```
+
+if none of `list|add|remove|sync` is specified, the command will default to `list`.
+
+## nuget trusted-signers list
```
Comment (rido-min) [inline docs/tools/cli-ref-trusted-signers.md]: Can we add a sample command output?

File: docs/reference/nuget-config-file.md (hunk 1 above)
Comment (PatoBeltran) [reply docs/reference/nuget-config-file.md]: But don't we want to give the user a sense of authors and repositories instead of just certificates? I like it better to talk about trusted authors and trusted repositories instead of certificates... certificates are just the means of trust, but technically what you want to trust is a given author or a given repository. 

Also, `require` mode by itself doesn't mean much to a new user that doesn't know that `trustedSigners` is related to `signatureValidationMode`

File: docs/reference/nuget-config-file.md (hunk 2 above)
Comment (PatoBeltran) [reply docs/reference/nuget-config-file.md]: This is just following the same schema that the previous sections in this example:

https://github.com/NuGet/docs.microsoft.com-nuget/pull/1156/files#diff-a0e3252e3f37fca1ac18e064c0032269R335

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3004?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3034.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3034.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3034?branch=pr-en-us-1156) |
[docs/reference/errors-and-warnings/NU3040.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/errors-and-warnings/NU3040.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu3040?branch=pr-en-us-1156) |
[docs/reference/nuget-config-file.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/reference/nuget-config-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-config-file?branch=pr-en-us-1156) |
[docs/tools/cli-ref-trusted-signers.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/pb-clientPolicies/docs/tools/cli-ref-trusted-signers.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/cli-ref-trusted-signers?branch=pr-en-us-1156) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811150057118316-1156%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KM1XmYu5u8JUmfYnhO2UWb6uFkFfQTwvQBJ6TsUmXXw%3D&st=2018-11-15T00%3A53%3A23Z&se=2018-12-16T00%3A58%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
